src/map/clif.c

```diff
@@ -8718,9 +8718,11 @@ void clif_charnameack (int fd, struct block_list *bl)
 				WBUFW(buf, 0) = cmd = 0x195;
 				memcpy(WBUFP(buf,6), ssd->fakename, NAME_LENGTH);
 				WBUFB(buf,30) = WBUFB(buf,54) = WBUFB(buf,78) = 0;
-				break;
+				if (!ssd->fakename_option)
```

Comment:
fakename_option is not bool, because this not allowed use !
if you want check it for 0, then write ``if (ssd->fakename_option == 0)``

Reply:
gud point. done.